.github/workflows/update-version.yml

```diff
@@ -0,0 +1,32 @@
+name: Update Version
+on:
+  workflow_dispatch:
+    inputs:
+      version: 
+        description: 'New version to use for the Edge Bridge extension. Example: 3.0.0'
+        required: true
+
+      core-dependency:
+        description: '[Optional] Update Core dependency in pom.xml. Example: 3.0.0'
```

Comment:
should pom.xml be gradle.properties ?

Reply:
This is consistent with our other extensions.
https://github.com/adobe/aepsdk-edge-android/blob/staging/.github/workflows/update-version.yml#L22
https://github.com/adobe/aepsdk-analytics-android/blob/staging/.github/workflows/update-version.yml#L22